src/article/index.scss

```diff
@@ -10,6 +10,61 @@ img {
   text-align: left;
 }
 
+.content-header::before {
+  @include settings.logo-img();
+  margin-top: 43px;
+}
+
+.content-header [data-itemprop~=authors] [itemtype~="http://schema.org/Person"] [itemprop~=familyName]:after {
+  content: ", ";
+}
+.content-header [data-itemprop~=authors] [itemtype~="http://schema.org/Person"]:last-child [itemprop~=familyName]:after {
+  content: "";
+}
+.content-header [data-itemprop~=authors] [data-itemprop~=affiliations] {
+  display: none;
+}
+.content-header [itemtype~="http://schema.org/Person"] [data-itemprop~=givenNames] [itemprop~=givenName]:after {
+  content: "\00a0";
+}
+.content-header [data-itemprop~=affiliations] [itemtype~="http://schema.org/Organization"]:after {
+  content: " • ";
+}
+.content-header [data-itemprop~=affiliations] [itemtype~="http://schema.org/Organization"]:last-child:after {
+  content: "";
+}
+
+
+.content-header [data-itemprop=about],
+.content-header [data-itemprop=genre],
+.content-header [data-itemprop~=identifiers],
+.content-header [data-itemprop~=keywords],
+.content-header [itemprop=datePublished] {
+  display: none;
+}
+
+.content-header [data-itemprop~=authors] {
+  list-style: none;
+  padding: 0;
+}
+.content-header [itemtype~="http://schema.org/Person"] {
+  @include typography.author-list-text();
+  display: inline;
+}
+
+.content-header [data-itemprop~=affiliations] {
+  list-style: none;
+  padding: 0;
+}
+.content-header [data-itemprop~=affiliations] [itemtype~="http://schema.org/Organization"] {
```

Comment:
Given we are extracting these can we remove the `data-itemprop` attributes and replace them with classes? It should also prevent some of the `Stencila` css from affecting the header.

Reply:
I hadn't planned on December ING, but yeah we should be able to relatively easily rewrap them. As you say it'll make the CSS much nicer (and frankly I think the header stuff is easily an early JSON-sourced improvement)